code/lib/core-server/src/utils/summarizeIndex.ts

```diff
@@ -6,11 +6,13 @@ const PAGE_REGEX = /(page|screen)/i;
 
 export const isPageStory = (storyId: string) => PAGE_REGEX.test(storyId);
 
+const isOnboardingEntry = (entry: IndexEntry) => ['example-button--warning'].includes(entry.id);
```

Comment:
would it make sense to check something like startsWith `example-button--` and does not match primary/secondary/large/small? If the user ends up creating 10 more stories in the example button, how do we want to count them?

Reply:
Good call!